### PR TITLE
Fix error message - rosa delete ocm-role

### PR DESCRIPTION
**Pull request by oriAdler**
In case the user deleted the role in their account via the AWS,
and they try to delete it with rosa-cli.
Prompt a warning to the user that the role doesn't exist on their account.

Deleting non-existing linked ocm-role:
image

Deleting non-existing unlinked ocm-role:
image

Related: [SDA-5689](https://issues.redhat.com/browse/SDA-5689)

### CODE REVIEW

**Comment by oriAdler** [comment]
@pvasant @zvikorn @zgalor @LeshemNoa PTAL

File: cmd/dlt/ocmrole/cmd.go [hunk 1]
```diff
@@ -219,36 +236,38 @@ func buildCommands(roleName string, roleARN string, isLinked bool, awsClient aws
...
+		hasPermissionBoundary, err := awsClient.HasPermissionsBoundary(roleName)
+		if err != nil {
+			return "", err
+		}
+		if hasPermissionBoundary {
+			deletePermissionBoundary := fmt.Sprintf("aws iam delete-role-permissions-boundary \\\n"+
+				"\t--role-name %s",
+				roleName)
+			commands = append(commands, deletePermissionBoundary)
+		}
```
Comment (LeshemNoa) [inline cmd/dlt/ocmrole/cmd.go:265]: I think the indentation here is incorrect.

**Comment by LeshemNoa** [comment]
LGTM!

File: cmd/dlt/ocmrole/cmd.go (hunk 1 above)
Comment (oriAdler) [reply cmd/dlt/ocmrole/cmd.go:265]: Seems like some error on Git:
image

**Pull request merged**